website/pages/en/index.js

```diff
@@ -13,6 +13,8 @@ const MarkdownBlock = CompLibrary.MarkdownBlock; /* Used to read markdown */
 const Container = CompLibrary.Container;
 const GridBlock = CompLibrary.GridBlock;
 
+import Members from "./member"; 
```

Comment:
Don't use import/export as we haven't set it up in our project. Use require instead
 
For projects like docusaurus, routing doesn't work this way. You need to look at the routing options.